charts/alpha/keycloak/values.yaml

```diff
@@ -0,0 +1,62 @@
+keycloak:
+  replicaCount: 1
+
+  auth:
+    existingSecret: ""
+    passwordSecretKey: password
+    adminUser: "admin"
+
+  networkPolicy:
+    enabled: false
+
+  postgresql:
+    enabled: false
+    primary:
+      networkPolicy:
+        enabled: false
+      containerSecurityContext:
+        enabled: false
+
+  externalDatabase:
+    existingSecret: "secret"
+    existingSecretDatabaseKey: ""
+    existingSecretHostKey: ""
+    existingSecretPasswordKey: ""
+    existingSecretPortKey: ""
+    existingSecretUserKey: ""
+
+  httpRelativePath: "/keycloak/"
+  containerSecurityContext:
+    enabled: false
```

Comment:
Why is this disabled? At the moment this fails helm-chart validation due to the security context.

Reply:
I had disabled this because the podsecuritycontext gets injected when deployed into rubix. I can explicitly add to avoid blocking the publish flow until we get alignment on the apollo-cli and rubix-admission though.